Source/MSBuildTools.Unity/Packages/com.microsoft.msbuildforunity/Editor/ProjectGenerator/Scripts/Utilities.cs

```diff
@@ -352,6 +375,17 @@ public static void CopyDirectory(string sourcePath, string destinationPath)
             //Copy all the files & Replaces any files with the same name
             foreach (string newPath in Directory.GetFiles(sourcePath, "*.*", SearchOption.AllDirectories))
             {
+                if (extensionFilters != null && extensionFilters.Length > 0)
+                {
+                    foreach (string extensionFilter in extensionFilters)
+                    {
+                        if (newPath.EndsWith(extensionFilter))
```

Comment:
>if (newPath.EndsWith(extensionFilter)) [](start = 24, length = 38)

`if (Path.GetExtension(newPath) == extensionFilter)`

Reply:
Well, I am fudging the meaning of extensions here a bit, as in ".cs.meta" being an extension, so doing it this way. I can rename?

Reply:
Oh I see. Maybe add a comment, and also should presumably be case-insensitive.